README.md

```diff
@@ -22,21 +22,529 @@ A Spezi framework that provides a common set of SwiftUI views and related functi
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct OnboardingView: View {
+    @State private var path = ManagedNavigationStack.Path()
+    
+    var body: some View {
+        ManagedNavigationStack(path: path) {
+            WelcomeStep()
+            ProfileStep()
+            NotificationStep()
+            CompletionStep()
+        }
+    }
+}
+
+private struct WelcomeStep: View {
+    @Environment(ManagedNavigationStack.Path.self) private var path
+    
+    var body: some View {
+        VStack {
+            Text("Welcome to the App!")
+            Button("Continue") {
+                path.nextStep()
+            }
+        }
+    }
+}
+```
+
+#### Share Sheet
+
+Easily share content with the system share sheet:
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct ShareView: View {
+    @State private var itemToShare: ShareSheetInput?
+    
+    var body: some View {
+        VStack {
+            Button("Share Text") {
+                itemToShare = ShareSheetInput("Hello World!")
+            }
+            
+            Button("Share Image") {
+                if let image = UIImage(systemName: "star") {
+                    itemToShare = ShareSheetInput(image)
+                }
+            }
+        }
+        .shareSheet(item: $itemToShare)
+    }
+}
+```
+
+#### Skeleton Loading
+
+Add skeleton loading animations:
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct LoadingView: View {
+    var body: some View {
+        VStack {
+            RoundedRectangle(cornerRadius: 10)
+                .fill(Color(.systemGray4))
+                .frame(height: 100)
+                .skeletonLoading(replicationCount: 5, repeatInterval: 1.5, spacing: 16)
+        }
+        .padding()
+    }
+}
+```
+
+### SpeziPersonalInfo - Personal Information
+
+Handle name input with proper formatting:
+
+```swift
+import SpeziPersonalInfo
+import SwiftUI
+
+struct ProfileSetupView: View {
+    @State private var name = PersonNameComponents()
+    
+    var body: some View {
+        Form {
+            Section("Personal Information") {
+                Grid(horizontalSpacing: 15) {
+                    NameFieldRow("First", name: $name, for: \.givenName) {
+                        Text("Enter your first name")
+                    }
+                    
+                    Divider()
+                        .gridCellUnsizedAxes(.horizontal)
+                    
+                    NameFieldRow("Middle", name: $name, for: \.middleName) {
+                        Text("Enter your middle name (optional)")
+                    }
+                    
+                    Divider()
+                        .gridCellUnsizedAxes(.horizontal)
+                    
+                    NameFieldRow("Last", name: $name, for: \.familyName) {
+                        Text("Enter your last name")
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+### SpeziValidation - Input Validation
```

Comment:
good to show a compact example here, might be nice to explain why this is important and what the target does. I am wondering if it rather first makes sense to explain the core concepts of `@ValidationState` and others here to piece things to gather and provide a motivation of the system architecture and the work @bauer-andreas did here?